src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -261,10 +266,10 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, scratchPool, layerThickness, laye
                   redi_term2(iTr, k, iCell) = redi_term2(iTr, k, iCell) - flux_term2*invAreaCell
                   redi_term2_edge(iTr, k, iEdge) = -flux_term2
 
-                  dTracerDx = (tracers(iTr, k, cell2) - tracers(iTr, k, cell1))*dvEdge(iEdge)*0.25_RKIND
+                  dTracerDx = (tracers(iTr, k, cell2) - tracers(iTr, k, cell1))/dcEdge(iEdge)*0.5_RKIND
                   fluxRediZTop(iTr, k) = fluxRediZTop(iTr, k) + &
                                          0.5_RKIND*(slopeTriadUp(k, iCellSelf, iEdge) + slopeTriadDown(k - 1, iCellSelf, iEdge)) &
-                                         *dTracerDx*invAreaCell
+                                         *dTracerDx
```

Comment:
This seems like a different bug fix, not mentioned in the PR itself, where the gradient was not being computed correctly?

Reply:
This is not correct as is.  It should be changed as well, but looking at this again it is not right. the top change here should be undone, but the second one is correct.  the top one is 
```
dTracerDx = (tracers(iTr, k, cell2) - tracers(iTr, k, cell1))/dcEdge(iEdge)**dvEdge(iEdge)*dcEdge(iEdge)*0.25_RKIND
```
So it does have the same averaging as the CVMix issues.  I will clean this file up